tools/install.py

```diff
@@ -176,6 +176,13 @@ def headers(action):
       'deps/zlib/zlib.h',
     ], 'include/node/')
 
+  # |icu_path| is set in small-icu and full-icu configurations
+  # but not when --with-intl= is set to none or system-icu.
+  icu_path = variables.get('icu_path')
+  if icu_path:
+    icu_headers_path = icu_path + '/source/common/unicode'
+    subdir_files(icu_headers_path, 'include/node/unicode/', action)
```

Comment:
This should probably also install i18n/unicode and (possibly?) io/unicode.

@srl295 What does ICU consider public API and what internal?